docs/source/federation-versions.mdx

```diff
@@ -0,0 +1,471 @@
+---
+title: Federation version changelog
+description: Understand changes between federation versions
+---
+
+## v2.3
+
+**Additions**
+
+[`@interfaceObject`](https://www.apollographql.com/docs/federation/federated-types/interfaces) a directive that allows you to abstract an interface in isolation by declaring it locally as an object type. This enables you to contribute fields without needing to redefine all the interface’s concrete implementation types.
+
+```graphql
+directive @interfaceObject on OBJECT
+```
+
+`@key` on interface definitions, in order to resolve the implementation type of the abstracted objects returned by the subgraphs using `@interfaceObject`.
+**Note:** This directive was already available in earlier versions of federation, but it wasn't used during composition.
```

Comment:
this directive already specified interface location in earlier version but it never worked (I think it was originally ignored but later on started to fail?)